web/beacon-app/src/features/members/utils.ts

```diff
@@ -1,6 +1,5 @@
 import { formatDate } from '@/utils/formatDate';
 export const formatMemberData = (m: any) => {
-  console.log('[formatMemberData] m: ', m);
```

Comment:
Would it be better to comment this out instead of deleting?

Reply:
I think yes